WIPv73Changelog.md

```diff
@@ -501,7 +606,26 @@ Default API version changed to "2022-07-28".
   | Person | `PersonParams.AddressKana` | `*AccountAddressParams` | `*PersonAddressKanaParams` | `AccountAddressParams` has been split into separate `PersonAddressKanaParams` and `PersonAddressKanjiParams` |
   | Person | `PersonParams.AddressKanji` | `*AccountAddressParams` | `*PersonAddressKanjiParams` | `AccountAddressParams` has been split into separate `PersonAddressKanaParams` and `PersonAddressKanjiParams` |
   | Plan | `Plan.AggregateUsage` | `string` | `PlanAggregateUsage` | Use enum that was already defined but unused |
-  | Plan | `Plan.TiersMode` | `string` | `PlanTiersMode` | Use enum that was already defined but unused | <!-- ---- R, S -->
+  | Plan | `Plan.TiersMode` | `string` | `PlanTiersMode` | Use enum that was already defined but unused |
+  | Refund | `Refund.SourceTransferReversal` | `*Reversal` | `*TransferReversal` | Rename `Reversal` to TransferRev`ersal for consistency with other Stripe client libraries |
+  | Refund | `Refund.TransferReversal` | `*Reversal` | `*TransferReversal` | Rename `Reversal` to TransferRev`ersal for consistency with other Stripe client libraries |
+  | SetupIntent | `SetupIntentPaymentMethodOptionsACSSDebit.Currency` | `string` | `*SetupIntentPaymentMethodOptionsACSSDebitCurrency` | `SetupIntentPaymentMethodOptionsACSSDebitCurrency` is an alias to `string` type |
+  | Source | `Source.Receiver` | `*ReceiverFlow` | `*SourceReceiver` | TODO |
+  | Source | `Source.Redirect` | `*RedirectFlow` | `*SourceRedirect` | TODO |
+  | Subscription | `SubscriptionListParams.Created` | `int64` | `*int64` | `Created` is nullable |
+  | Subscription | `SubscriptionListParams.Customer` | `string` | `*string` | `Customer` is nullable |
+  | Subscription | `SubscriptionListParams.Plan` | `string` | `*string` | `Plan` is nullable |
+  | Subscription | `SubscriptionListParams.Price` | `string` | `*string` | `Price` is nullable |
+  | Subscription | `SubscriptionListParams.Status` | `string` | `*string` | `Status` is nullable |
```

Comment:
Sorry I was wrong about these. The rationale is that Params fields should be pointers